pallets/tcr/src/lib.rs

```diff
@@ -21,8 +21,9 @@
 //! This module implements a Token Curated Registry where members (represented by their
 //! `AccountId`) are accepted based on the number of tokens staked in support to their
 //! application.
-
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
do you need that flag? It is normally showing up on top of the `benchmarking.rs` file

Reply:
Indeed not required, removed, Done :+1: